src/Modules/Authentication/Register/Register.tsx

```diff
@@ -1,7 +1,255 @@
+              })}
+            />
+             {errors?.country && <p style={{color:"red",fontSize:'12px'}}>{String(errors?.country?.message)}</p>}
+             </Stack>
+            </Stack>
+            {/* email  */}
+
+            <TextField id="outlined-basic" 
+            label="Email Address" variant="outlined" 
+            placeholder='please type here...' 
+            sx={{background: "#F5F6F8"}}
+            {...register('email', EMAIL_VALIDATION)}
+             />
+             {errors?.email && <p style={{color:"red",fontSize:'12px'}}>{String(errors?.email?.message)}</p>}
+
+            {/* password */}
+           <div className={styles["password"]}>
+            <TextField id="outlined-basic" 
+            label="Password" variant="outlined" 
+            placeholder='please type here...'
+            type={typePassword} 
+            sx={{background: "#F5F6F8"}}
+            {...register('password', {
+              required: 'Password is required'
+            })}
+             />
+             {typePassword === 'password' ? <button type='button' onClick={typeChangePassword}>
+                  {/* <i className="fa-solid fa-eye-slash"></i> */}
+                  <VisibilityOffIcon/>
+                </button> : <button type='button' onClick={typeChangePassword}><RemoveRedEyeIcon/></button>}
+            </div>    
+            {errors?.password && <p style={{color:"red",fontSize:'12px'}}>{String(errors?.password?.message)}</p>}
+            {/* confirmPassword  */}
+
+           <div className={styles["password"]}>
+            <TextField id="outlined-basic" 
+            label="Confirm Password" 
+            variant="outlined" 
+            placeholder='please type here...' 
+            sx={{background: "#F5F6F8"}}
+            {...register('confirmPassword', {
+              required: 'Confirm password is required',
+              validate: (confirmPassword) =>
+                confirmPassword === watch('password') || "Password do not match",
+            })}
+             />
+             {typeConfirmPassword === 'password' ? <button type='button' onClick={typeChangeConfirmPassword}>
+                  {/* <i className="fa-solid fa-eye-slash"></i> */}
+                  <VisibilityOffIcon/>
+                </button> : <button type='button' onClick={typeChangeConfirmPassword}><RemoveRedEyeIcon/></button>}
+              </div>   
+            {errors?.confirmPassword && <p style={{color:"red",fontSize:'12px'}}>{String(errors?.confirmPassword?.message)}</p>}
+
+            
+          <input type="file" 
+            {...register('profileImage')}
+            
+           />
+            <Button type='submit' variant="contained" size="large" disabled={isSubmitting}>
+              {isSubmitting? "sign up..." : "sign up"}
+            </Button>
+          </Stack>
+          
+        </Stack>
+        <div
+          style={{
+            width: "50%",
+          }}
+        >
+          <img
```

Comment:
Instead of creating the layout in each page. We could create an AuthLayout component that will take the image as a prop and the form as a children. This will make our code more consistent. We can also make a shared TextField component with the styles that we want and re-use it in other screens.